packages/persona/src/types/talent.ts

```diff
@@ -0,0 +1,75 @@
+// TalentProtocol API Types
+export type TalentAccountSource = 'wallet' | 'farcaster' | 'github';
+
+export interface TalentQueryParams {
+  id: string; // Talent ID, wallet address or account identifier
```

Comment:
```suggestion
  /** Talent ID, wallet address or account identifier */
  id: string; 
```

Current comment does not have any effect to inline suggestions. Please fix for other comments as well.